src/components/flat-table/flat-table-row/flat-table-row-draggable.component.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
```

Comment:
This should be in an internal folder as consumers will never import it directly, then you can remove the private/ignores from the prop types 

It also needs a TypeScript definition file